pkg/cli/cli.go

```diff
@@ -200,15 +200,7 @@ func (c *CLIRunner) DeleteACL(
 
 // BootstrapTopics creates configs for one or more topics based on their current state in the
 // cluster.
-func (c *CLIRunner) BootstrapTopics(
-	ctx context.Context,
-	topics []string,
-	clusterConfig config.ClusterConfig,
-	matchRegexpStr string,
-	excludeRegexpStr string,
-	outputDir string,
-	overwrite bool,
-) error {
+func (c *CLIRunner) BootstrapTopics(ctx context.Context, topics []string, clusterConfig config.ClusterConfig, matchRegexpStr string, excludeRegexpStr string, outputDir string, overwrite bool, allowUnderscoreTopics bool) error {
```

Comment:
Can you reformat this to multiple lines for each param? It's much easier to read that way

Reply:
fixed in 4e53e18